PLAT-121194: ThemeDecorator -  Added 'enact-fit' class based on 'disableFullscreen' prop (#502)
enactjs/agate: CHANGELOG.md

- `agate/Popup` content padding and margin with `closeButton`
- `agate/Popup` `bottom`, `fullscreen`, `left`, and `right` values to `position` prop
- `agate/Popup` `duration` and `type` props

> - `agate/ThemeDecorator` class `enact-fit` based on the value of `disableFullScreen` prop

Reviewer: ```suggestion
- `agate/ThemeDecorator` to exclude `enact-fit` className when `disableFullscreen` is true
```
What about this?

It looks better to move this under ### Changed. 🤔
